doc/source/support/frequently-asked-questions.rst

```diff
@@ -351,18 +348,37 @@ Visit the following URL of your package: ``https://github.com/<org-name>/<packag
 
 Click ``Run workflow`` and select the ``main`` branch. Your online documentation will be updated with the latest changes without a new release.
 
+.. _faq-doc-favicon-logo:
+
+How do I add a favicon and logo to the documentation?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In ``doc/source/conf.py``, add the following lines:
+
+.. code-block:: python
+
+  html_theme_options = {
+    "navigation_with_keys": True,
+    "logo_only": True,
+  }
+
+  html_favicon = "<path-to-favicon>"
+  html_logo = "<path-to-logo>"
+
+The clickable logo will be displayed above the menu bar on the left side of the page.
+
 conda-forge
 -----------
 
 How do I add a new admin to the conda-forge feedstock?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Please refer to the admin section in the conda-forge release guide :ref:`here <conda-forge-add-admin>`.
```

Comment:
sphinx doens't like `:ref:here` if it is used more than once. Also, instead of using "here", I realized it's much better to state that "here" is with the URL attached.